src/Symfony/Bundle/FrameworkBundle/Messenger/RouterContextMiddleware.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Messenger;
```

Comment:
shouldn't this be in the Messenger component?

Reply:
moved.